src/components/navbar/SealVerse.tsx

```diff
@@ -0,0 +1,10 @@
+import Dropdown from 'components/Dropdown'
+
+export default function () {
+  const options = [
+    { label: 'SealCred', selected: true },
+    { label: 'SealCred Echo', href: 'https://echo.sealcred.xyz' },
+  ]
```

Comment:
Maybe we can put it into the config like `data`? Or include it in the existing file? Wdyt?